aten/src/ATen/native/TensorShape.cpp

```diff
@@ -847,6 +847,11 @@ Tensor expand_symint(const Tensor& self, c10::SymIntArrayRef packed_size, bool i
   return expand(self, size, implicit);
 }
 
+Tensor view_symint(const Tensor& self, c10::SymIntArrayRef size) {
+  auto size_ = asIntArrayRefSlow(size);
+  return self.view(size_);
```

Comment:
oh btw @Krovatkin, I'm just thinking - I think that for dynamo (in general when we do tracing), this implementation should actually be totally fine. `view.symint` and `view_copy.symint` are `CompositeExplicitAutograd` ops, which means that they get treated as "primitives" when we trace. So these kernels don't actually run - we just trace the `view.symint` or `view_copy.symint` directly into the graph.


It'll probably be a problem for LTC/XLA though: we don't want these kernels to run. For the view/view_copy ops, that'll actually be easy - we can use the `CompositeExplicitAutogradNonFunctional` alias key to opt XLA/LTC out of running these kernels.

The only other case I think we might have to worry about is existing composite operators that need symint versions. We don't want them to call their concrete-int variants... I guess we can just take the existing de-composition for the non-symint version, and duplicate it for the symint version? Probably depends how many composite ops there are that take in `int[]/int` args though.